pwiz_tools/Skyline/FileUI/PeptideSearch/BuildPeptideSearchLibraryControl.cs

```diff
@@ -250,16 +250,41 @@ private void listSearchFiles_SelectedIndexChanged(object sender, EventArgs e)
 
         private void btnAddFile_Click(object sender, EventArgs e)
         {
+          if (PerformDDASearch)
+            {
+                
+                using (OpenFileDialog dlg = new OpenFileDialog
+                {
+                    //todo add resources here
+                    Title = "Add spectra files to analyze",
+                    InitialDirectory = Path.GetDirectoryName(DocumentContainer.DocumentFilePath),
+                    CheckPathExists = true,
+                    Multiselect = true,
+                    Filter = "spectrum files(*.mgf;*.MGF;*.mzML;*.mzml;*.MZML; *.raw)|*.mgf;*.MGF;*.mzML;*.mzml;*.MZML;*.raw"

```

Comment:
This should be search-engine dependent, or we can add a user-transparent msconvert step if the engine does not support the given file type.

Reply:
NB: Brendan suggests the transparent msconvert step. Fortunately not needed for MSAmanda.